setup.py

```diff
@@ -18,6 +18,7 @@
         "setuptools>=41.4.0",
         "pandas>=1.0.3",
         "coverage>=5.5",
-        "PyMuPDF==1.18.13"
+        "PyMuPDF==1.18.13",
+        "python-magic>=0.4.24"
```

Comment:
If you are using new pip installed libraries, make sure they are added to the setup.py and pyproject.toml (only pyproject.toml actually determines what gets installed as dependencies via pip)